src/NUnitConsole/nunit3-console/ConsoleOptions.cs

```diff
@@ -67,6 +67,8 @@ public class ConsoleOptions : CommandLineOptions
 
         public bool LoadUserProfile { get; private set; }
 
+        public string ConsoleEncoding { get; private set; }
```

Comment:
Ironically, that is where I had originally placed it, but I thought ConsoleOptions was more closely related. Will move.